app/(non-authorized)/_layout.tsx

```
@@ -1,38 +1,32 @@
-import {
-  DrawerContentScrollView,
-  DrawerItemList,
-} from "@react-navigation/drawer";
-import { Redirect } from "expo-router";
-import { Drawer } from "expo-router/drawer";
+import { Redirect, Stack } from "expo-router";
 import { FC } from "react";
 
 type NonAuthorizedLayoutPropsType = {};
 
 const NonAuthorizedLayout: FC<NonAuthorizedLayoutPropsType> = ({}) => {
```

Biome

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)